front/src/modules/timesStops/TimesStops.tsx

```diff
@@ -79,50 +40,18 @@ const TimesStops = ({
       className="time-stops-datasheet"
       columns={columns}
       value={rows}
-      onChange={(newRows: PathWaypointRow[], [op]) => {
-        if (!isInputTable) {
-          return;
-        }
-        let updatedRows = [...newRows];
-        updatedRows[op.fromRowIndex] = updateRowTimesAndMargin(
-          newRows[op.fromRowIndex],
-          rows[op.fromRowIndex],
-          op,
-          allWaypoints.length
-        );
-        updatedRows = updateDaySinceDeparture(updatedRows, startTime);
-        if (!updatedRows[op.fromRowIndex].isMarginValid) {
-          newRows[op.fromRowIndex].isMarginValid = false;
-          setRows(newRows);
-        } else if (
-          !rows[op.fromRowIndex].isMarginValid &&
-          updatedRows[op.fromRowIndex].isMarginValid
-        ) {
-          newRows[op.fromRowIndex].isMarginValid = true;
-          setRows(newRows);
-        } else {
-          const newVias = updatedRows
-            .filter((row, index) => !isEqual(row, rows[index]))
-            .map(
-              (row) =>
-                ({
-                  ...row,
-                  ...(row.arrival && { arrival: durationSinceStartTime(startTime, row.arrival) }),
-                }) as SuggestedOP
-            );
-          dispatch(upsertSeveralViasFromSuggestedOP(newVias));
+      onChange={(newRows: T[], [op]) => {
+        if (onChange) {
+          onChange(newRows, op);
         }
       }}
       stickyRightColumn={stickyRightColumn}
       lockRows
       height={600}
       headerRowHeight={headerRowHeight}
-      rowClassName={({ rowData, rowIndex }) =>
+      rowClassName={({ rowData }) =>
         cx({
-          activeRow:
-            rowIndex === 0 ||
-            rowIndex === allWaypoints.length - 1 ||
-            isVia(pathSteps || [], rowData, { withKP: true }),
+          activeRow: rowData.isWaypoint,
```

Comment:
Discussed, we can accept this small regression since it's only for imported trains. I will open a issue to manage more globally the imported trains (their vias are not in bold right now on dev)